src/Database/LSMTree/Internal/Index/Compact.hs

```diff
@@ -378,11 +376,6 @@ instance NFData IndexCompact where
   Queries
 -------------------------------------------------------------------------------}
 
--- |  Searches for a page span that contains a key–value pair with the given key.
---
--- If there is indeed such a pair, the result is the corresponding page span; if
--- there is no such pair, the result is an arbitrary but valid page span.
---
```

Comment:
If the idea here is that we can remove the comment because the documentation is now elsewhere (i.e., the `Index` class), then we should link to that documentation here